content/en/news/releases/1.13.x/announcing-1.13/upgrade-notes/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Upgrade Notes
+description: Important changes to consider when upgrading to Istio 1.13.0.
+weight: 20
+---
+
+{{< warning >}}
+This is an automatically generated rough draft of the release notes and has not yet been reviewed.
+{{< /warning >}}
+
+When you upgrade from Istio 1.12.x to Istio 1.13.0, you need to consider the changes on this page.
+These notes detail the changes which purposefully break backwards compatibility with Istio 1.13.0.
+The notes also mention changes which preserve backwards compatibility while introducing new behavior.
+Changes are only included if the new behavior would be unexpected to a user of Istio 1.12.x.
+
+## Health Probes will no longer re-use connections
+
+Health probes using the istio-agent [health probe rewrite](https://istio.io/latest/docs/ops/configuration/mesh/app-health-check/) will
+now no longer re-use connections for the probe. This behavior was changed to match probing behavior of Kubernetes',
+and may also improve probe reliability for applications using short idle timeouts.
+
+As a result, your application may see more connections (but the same number of HTTP requests) from probes.
+For most applications, this will not be noticeably different.
```

Comment:
Would be good to notify users what type of applications this may affect.

Reply:
I cannot think of any possible application that this would impact - this is more of a CYA statement.